emails/templates/generic.js

```diff
@@ -100,13 +100,13 @@ module.exports = function() {
         Have fun, and break things!
       `,
       notice: `
-        This is an email from <strong>Aragon Court Rinkeby test environment</strong>. This
+        This is an email from <strong>Aragon Court's Rinkeby test environment</strong>. This
```

Comment:
Since this is going into the html, should we use the fancy apostrophe here @bpierre?

Reply:
Yes, I hope every email client supports UTF-8 now :laughing:

Reply:
https://github.com/aragon/court-dashboard/pull/142/commits/dc3e89ff2370141173ddf2f2c3303f3527488670